src/Command/VersionCommand.php

```diff
@@ -57,6 +57,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 
     $io = new DrupalStyle($input, $output);
     $io->info($this->toSemanticVersion($info['version']));
+    $io->warning('This command is deprecated and will be removed in a future version of Lightning. Please use the drush command \'drush lightning:version\' instead.');
```

Comment:
Nice. Let's rephrase this slightly; how about "This command is deprecated and will be removed in Lightning 4.2.0. Please use the Drush command `drush lightning:version` instead." Also, let's output the warning _before_ the version number.